lib/OpenQA/Setup.pm

```diff
@@ -100,6 +100,7 @@ sub read_config ($app) {
             require_for_assets => 0,
         },
         'scm git' => {
+            git_auto_commit => 'yes',
```

Comment:
would you mind to move this along with the other git_auto_* down in the L108?

Reply:
I intentionally put it here because it is the precursor to the following options - `update_remote`, `update_branch`, `do_push` and `do_cleanup` all configure specific aspects of the `git_auto_commit` behaviour. To me it makes more sense to organize the options by what they do than by what they're called.